### PR TITLE
Update personas document for issue #3100

### PR DESCRIPTION
**Pull request by victoriaatraft**
## Summary of Changes
Updated OFA analysts description tables and added OFA Data Analyst Persona
Pull request for https://github.com/raft-tech/TANF-app/issues/3100

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.
> *Demo GIF(s) and screenshots for testing procedure*

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [ ] [**_insert ACs here_**]
+ [ ] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [ ] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [ ] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [ ] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

**Review by reitermb — APPROVED** [review]
Final tweaks discussed in internal UX sync, good to QASP!

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 1]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+
+### Elliott — OFA System Admin
+
+| Grace                               |                                                              |
+| :---------------------------------- | :----------------------------------------------------------- |
+| Role                                | OFA Data Portal User, Viewing and Editing Permissions        |
+| Age                                 | 37                                                           |
+| Location                            | USA, Occassionally In Office DC                              |
+| Work Environment                    | Home and Office, Desktop, and Government Furnished Equipment (GFE) |
+| Goals<br /><br /><br />             | 1. Ensure availability of features and components of the system <br />2. Work with regional OFA staff to confirm who should and shouldn't be granted access to the system<br />3. Access system logs to troubleshoot problems and remediate scans as needed <br />4. Provide access to OCIO staff for security auditing purposes <br />5. Monitoring emails, notifications and answering STT Questions |
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
| Goals<br /><br /><br />             | 1. Ensure availability of features and components of the system <br />2. Work with regional OFA staff to confirm who should and shouldn't be granted access to the system<br />3. Access system logs to troubleshoot problems and remediate scans as needed <br />4. Provide access to OCIO (ACF-Tech) staff for security auditing purposes <br />5. Monitoring emails, notifications and answering STT Questions |
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 2]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+| Pain Points<br /><br /><br /><br /> | 1. "Django doesn't have all the filters I need to efficiently use it for certain admin tasks" <br /> 2. "The system can sometimes log me out with no warning" <br/> 3. Accessing data in relational data bases is only possible by disconnecting the HHS network on site. <br/> 4. The general toolstack as an administrator can be difficult to use on a GFE <br/> 5. Requesting access to developer tools can take months at a time to get approved                           |
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
| Pain Points<br /><br /><br /><br /> | 1. "Django doesn't have all the filters I need to efficiently use it for certain admin tasks" <br /> 2. "The system can sometimes log me out with no warning" <br/> 3. Accessing data from cloud.gov-brokered databases is only possible by disconnecting from the HHS network on site and VPN when remote. <br/> 4. The toolstack needed as an administrator can be difficult to use on a Windows GFE <br/> 5. Requesting access to developer tools can take months at a time to get approved                           |
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 3]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+| Delights                            | 1. "Django has become a tool that provides me a lot of insights about data in the system rather than solely being a user administration tool" <br/> 2. "Automated notifications have been convenient [so that I am aware of what is happening]" <br/> 3. "Being able to use an interface and see what [actions] users did" <br/> 4. "A point and click experience to see how the system is performing...I use the front end the most"|
+| Tools                               | TDP Portal, Django CSV exports, Excel, SAS, email, reporting/analytics software, Microsoft Suite, Python, Axe Dev Tools |
+| Expectations | 1. TDP should help sys admin do things from an interface as much as possible <br /> 2. We have to have another way to troubleshoot and remediate issues |
```
Comment (ADPennington) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
| Expectations | 1. TDP should help sys admin do things from an interface as much as possible <br /> 2. We need to have multiple ways to troubleshoot and remediate issues |
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
in pretty good shape! a few edit suggestions

**Review by ADPennington — APPROVED** [review]
lgtm @victoriaatraft 👍🏾 tagging @ttran-hub here for review because i wasnt able add him as a reviewer on this PR for some reason.

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 4]
```diff
@@ -82,15 +82,15 @@ Some grantees segment data preparation and data transmission into two distinct r
 
 ### OFA analysts
 
-OFA analysts review, analyze, and report on STT’s TANF data that’s submitted through TDP. They also help STTs through the transmission process by offering support and guidance around data quality (including fatal/warning edits). Analysts in the Tribal TANF Division also play a direct role communicating submission status, resubmission requests, and various reports to Tribes.
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md:85]: Please delete "(including fatal/warning edits)".

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 5]
```diff
@@ -82,15 +82,15 @@ Some grantees segment data preparation and data transmission into two distinct r
...
 **Supporting Documentation**
 
 - [Description of the OFA support mailbox](<https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA-UserResearch/Shared%20Documents/User%20Research/OFA%20Resource%20Mailbox%20notes.docx?d=w2542a110040b44c1b82706a54c6ec9f2&csf=1&web=1&e=O5meXQ>) :lock:
 
 ### OFA system admins
 
-OFA admins are members of the TANF Data Division. They oversee the collection and processing of TANF data and ensure OFA analysts have what they need to do their work.
+OFA admins are members of the TANF Data Division. They manage access requests for the TANF data portal, help troubleshoot issues with submission, and broadly ensure that OFA analysts have what they need to do their work.
 
 ### Regional program specialists
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md:96]: Please change grantee to STT under regional program specialist

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 6]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+| Delights                            | 1. "With TDP, I have a lot [of data] once the State submits the data the data gets processed already so that's really good. Now  I don't need to spend too much time with state if the  data wasn't processed." <br /> 2. "Having direct and up to date access to the data in the database" |
+| Tools                               | Django CSV exports, Excel, SAS, email, reporting/analytics software |
+
+
+### Elliott — OFA System Admin
+
+| Grace                               |                                                              |
+| :---------------------------------- | :----------------------------------------------------------- |
+| Role                                | OFA Data Portal User, Viewing and Editing Permissions        |
+| Age                                 | 37                                                           |
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]: Can Elliott be 46 or something?

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 7]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
-| Pain Points<br /><br /><br /><br /> | 1. Transmission reports can be confusing and aren't actionable enough<br />2. Current errors aren't easy to understand and act to fix, nor easy to explain to grantees<br />3. It can be difficult to identify where an error is coming from in a file<br />4. I see submitted data differently (in different systems) than the grantee submitting it does—The lack of a shared place to view data complicates communication.<br /><br /> |
-| Delights                            | 1. We send out WPR reports far faster than we used to        |
-| Tools                               | OCIO database, scripts to interact with or query the database, email, reporting/analytics software, excel, fTANF.exe |
+| Work Environment                    | Home and Office, Office                                      |
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
| Work Environment                    | Home and Office|
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 8]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+| Tools                               | Django CSV exports, Excel, SAS, email, reporting/analytics software |
+
+
+### Elliott — OFA System Admin
+
+| Grace                               |                                                              |
+| :---------------------------------- | :----------------------------------------------------------- |
+| Role                                | OFA Data Portal User, Viewing and Editing Permissions        |
+| Age                                 | 37                                                           |
+| Location                            | USA, Occassionally In Office DC                              |
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
| Location                            | USA, Occasionally In DC Office                              |
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 9]
```diff
@@ -109,7 +109,7 @@ respect to TDP, OCIO staff will have access to subsets of the Django Admin exper
 
 ## Personas
 
-Research up to this point has suggested a need for two working personas; an OFA Data Portal User (Grace) and a grantee-side persona. While there's no shortage of variety in roles, responsibilities, and team makeups on both the grantee and federal levels, the "jobs to be done" for each in the TANF Data Portal are shared. A single persona could also have many different combinations of permissions available to them in the data portal. e.g. While the OFA Persona represents all tasks that an OFA user might want to complete, some may only be assigned to some subset of those tasks. 
+Research up to this point has suggested a need for three working personas; an OFA Data Analyst (Grace), an OFA System Admin (Elliott), and an STT data analyst. While these don't reflect the full diversity of  roles, responsibilities, and team makeups on both the grantee and federal levels, the "jobs to be done" for each in the context of TDP have a lot of overlap. A single persona could also have many different combinations of permissions available to them in the data portal.
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
Research up to this point has suggested a need for three working personas: an OFA Data Analyst (Grace), an OFA System Admin (Elliott), and an STT data analyst. While these don't reflect the full diversity of  roles, responsibilities, and team makeups on both the grantee and federal levels, the "jobs to be done" for each in the context of TDP have a lot of overlap. A single persona could also have many different combinations of permissions available to them in the data portal. 
```

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 10]
```diff
@@ -118,11 +118,28 @@ Research up to this point has suggested a need for two working personas; an OFA
...
+| Delights                            | 1. "Django has become a tool that provides me a lot of insights about data in the system rather than solely being a user administration tool" <br/> 2. "Automated notifications have been convenient [so that I am aware of what is happening]" <br/> 3. "Being able to use an interface and see what [actions] users did" <br/> 4. "A point and click experience to see how the system is performing...I use the front end the most"|
+| Tools                               | TDP Portal, Django CSV exports, Excel, SAS, email, reporting/analytics software, Microsoft Suite, Python, Axe Dev Tools |
+| Expectations | 1. TDP should help sys admin do things from an interface as much as possible <br /> 2. We need to have multiple ways to troubleshoot and remediate issues |
+
+To view a further in depth system of the interactions, primary tasks and touchpoints taken by OFA System Admins, please refer to the ecosystem map in the supporting documents below.
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]:
```suggestion
To view a further in-depth system of the interactions, primary tasks and touchpoints taken by OFA System Admins, please refer to the ecosystem map in the supporting documents below.
```

**Review by lfrohlich — CHANGES_REQUESTED** [review]
Looking good! Made some minor comments

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md (hunk 6 above)
Comment (victoriaatraft) [reply docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]: updated

File: docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md [hunk 11]
```diff
@@ -141,3 +158,4 @@ Research up to this point has suggested a need for two working personas; an OFA
 
 - [Grantee Attributes](<https://hhsgov.sharepoint.com/:x:/r/sites/TANFDataPortalOFA-UserResearch/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Be83f84f1-71a0-459f-80a4-0d39bc250be9%7D&action=view&cid=f2e656cb-9245-4f02-af6d-2d7e1db6573d>) :lock:
 - [Working Journey Maps & Personas Mural](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1608238114372/191c53b8ef538838bc8c179daa238dd5c5dcc9e8) :lock:
+- [Ecosystem Map OFA System Admin](https://app.mural.co/t/raft2792/m/raft2792/1728311825176/74942929a28857c3ef1c401f0d07ccfe1073e882?sender=uf443e00cb4626b8bec157528)
```
Comment (lfrohlich) [inline docs/User-Experience/Research-Syntheses/2020, Summer - Understanding Stakeholders and creating personas.md]: Can you add the :lock: icon to the last link? 
```suggestion
- [Ecosystem Map OFA System Admin](https://app.mural.co/t/raft2792/m/raft2792/1728311825176/74942929a28857c3ef1c401f0d07ccfe1073e882?sender=uf443e00cb4626b8bec157528) :lock:
```

**Review by lfrohlich — CHANGES_REQUESTED** [review]
One minor thing, then will be all good to go!

**Comment by reitermb** [comment]
> One minor thing, then will be all good to go!

Committed! Back over to QASP

**Review by lfrohlich — APPROVED** [review]
LGTM

**Pull request merged**